lib/train/transports/vmware.rb

```diff
@@ -0,0 +1,190 @@
+# encoding: utf-8
```

Comment:
That said, pleas update the README.md to include vmware, and provide configuration information there.

Reply:
I've added sections covering basic usage. Detailed info (such as PWSH/PowerCLI installation will be in the resource pack).

Does that work for you? 😄